bin/windows-env.ps1

```diff
@@ -49,9 +55,8 @@ $Wix35_VERSION = '3.5.2519.20130612'
 $env:PATH = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
 if ($arch -eq 32) {
   $env:PATH = "C:\tools\mingw32\bin;" + $env:PATH
+} else {
+  $env:PATH = "C:\tools\mingw64\bin;" + $env:PATH
 }
-$env:PATH += [Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd"
+$env:PATH += [Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd" + ";" + "$toolsDir\$rubyPkg\bin" + ";" + "$toolsDir\$opensslPkg\bin"
 Write-Host "Updated Path to $env:PATH"
-
-# SSL root pointer.
```

Comment:
I assume `OPENSSL_ROOT_DIR` is being removed because we're using pre-built packages that have already had this set during compilation?

Should probably note that in commit message if that's the case.

I asked about this variable when it was originally added... and was told that it was necessary -- see https://github.com/puppetlabs/puppet-agent/pull/311#discussion_r40980262